docs/source/conf.py

```diff
@@ -179,6 +179,7 @@
 # documentation for each type specified
 # The following elements are the link that auto doc were not able to do
 nitpick_ignore = [
+    ("py:class", "pd.Series"),
     ("py:class", "ndarray"),
     ("py:class", "pydantic.main.BaseModel"),
     ("py:class", "torch.nn.modules.module.Module"),
```

Comment:
We should delete everything related to pytorch (lines 184-189) since it is not used in PyDESeq2

```suggestion
```